custom/icds_reports/ucr/data_sources/awc_mgt_forms.json

```diff
@@ -178,6 +178,23 @@
         "column_id": "awc_location",
         "type": "expression"
       },
+      {
+        "display_name": null,
+        "column_id": "location_entered",
+        "type": "boolean",
+        "filter": {
+          "type": "not",
+          "filter": {
+          "type": "boolean_expression",
+          "operator" : "in",
+          "expression": {
+	        "type": "property_path",
+	        "property_path": ["form", "location_form_filled"]
```

Comment:
I think it would be better to save the string value of `location_form_filled` and `meeting_type` instead of a boolean of whether or not they were entered. It would make future changes easier.

Reply:
Hmm. Thats a good thought. I made it boolean because we don't need the value right now. But yeah, taking the simple value will also work. we can ignore if the value if `null` or `''`.